example_node.js

```diff
@@ -40,8 +41,9 @@ dfu.findDevice({ services: [0x180D] })
     var hex = file.toString();
     var buffer = hex2bin(hex);
     log("file length: " + buffer.byteLength);
+    var crc = crc16(buffer);
 
-    return dfu.provision(device, buffer);
+    return dfu.provision(device, buffer, 4, crc);
```

Comment:
If we swap the args (above), this negates the need for the '4' here. If we aren't swapping them, you could use ```dfu.ImageType.Application``` instead